doc/examples.xml

```diff
@@ -703,3 +703,45 @@ true
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="BookDigraph">
+<ManSection>
+  <Oper Name="BookDigraph" Arg="[filt, ]m"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    The <E>Book digraph</E> is (generally) the cartesian product of the chain digraph
+    (as the "Book Spine") and the star digraph (as the "Pages"). When <A>m</A> is less than 3
+    two special cases occur. When <A>m</A> is 1, a (<A>2</A>, <A>2</A>) complete bipartite
+    graph is returned (the "square" graph). When <A>m</A> is 2, a (<A>2</A>, <A>3</A>)
+    square grid graph is returned (the "domino" graph). For more details on the Book digraph
+    please refer to <URL>https://mathworld.wolfram.com/BookGraph.html</URL>.<P/>
```

Comment:
* In the Digraphs package we seem to capitalise the word 'Cartesian' - see e.g. the documentation for `DigraphCartesianProduct`.
* The Cartesian product of _which_ chain digraph and _which_ star digraph? (i.e. on how many vertices, respectively?) I think this should be documented.
  * (From looking at the link, the numbers are `2` and `<M><A>m</A>+1</M>`, respectively.)
* In fact, after looking at the link, I noticed that it's not the chain digraph that's involved! The chain digraph isn't a symmetric digraph; in particular, `ChainDigraph(2)` just consists of the one edge `1 → 2`. You should define it (and code it) in terms of either the complete digraph on two vertices, or the cycle digraph on two vertices (which are equal).
* I think the information about the 'special cases' is misleading. When I first read this documentation, I assumed that these 'special cases' don't fit the general pattern of being the Cartesian product of the complete digraph on 2 vertices with the star digraph on m+1 vertices. But in fact they do fit in this pattern. So they don't need to be defined separately.
    * If you really want to include this information, I think it should be a separate paragraph, and it should be clear that this is additional information given for interest, not part of the definition. (Something along the lines of "Note that `BookDigraph(1)` is a (2,2)-complete bipartite digraph, and `BookDigraph(2)` is isomorphic to the (2,3)-square grid graph"). But I would recommend just not including this.
    * Note that, in your implementation at least, `BookDigraph(2) <> SquareGridGraph(2,3)`, although they are isomorphic.